Source/Management.GraphQL/EventHandlers/EventHandler.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dolittle.Runtime.Management.GraphQL.EventHandlers
+{
+    public class EventHandler
+    {
+        public Guid Id { get; set; }
+        public Guid Scope { get; set; }
+        public int FilterPosition { get; set; }
+        public int EventProcessorPosition { get; set; }
+        public int TailEventLogSequenceNumber { get; set; }
+    }
+}
```

Comment:
It seems to me like this does not account for partitioned event handlers? Not sure if we need to now if the UI doesn't show failures or positions per partition.